util/SerializeMultiplayerCommon.cpp

```diff
@@ -15,8 +16,14 @@
 template <class Archive>
 void GalaxySetupData::serialize(Archive& ar, const unsigned int version)
 {
-    ar  & BOOST_SERIALIZATION_NVP(m_seed)
-        & BOOST_SERIALIZATION_NVP(m_size)
+    if (Archive::is_saving::value && m_encoding_empire != ALL_EMPIRES && (!GetOptionsDB().Get<bool>("network.server.publish-seed"))) {
+        std::string dummy = "hidden";
```

Comment:
Is this "hidden" text meant to be shown to the player on the galaxy info pedia page or multiplayer lobby? If so, there shouldn't be localized text in the source code...

Reply:
I'm not sure what to place here as any localization will happen on the server side with some default language instead of client side. Maybe send "-" or some sign to the client?

Reply:
Unless an empty string has already another special meaning use that and let the client handle the seed representation.

Reply:
In the client UI, setting an empty seed sets it to generate a random seed. The universe generation also treats an empty seed as a sign to generate a random one server-side. But neither of those mean an empty seed can't be sent back to the client later...

Reply:
So I suppose empty seed is fine to send to the client.